aries-site/src/examples/templates/filtering/FilteringLists/FilteringLists.js

```diff
@@ -88,17 +96,24 @@ export const FilteringLists = ({ containerRef }) => {
           Orders
         </Heading>
       </Header>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={orders}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
```

Comment:
Do we have to add this Box with `fill` or can we just add `fill` directly on Data?